src/objects/entities/Ben3.ts

```diff
@@ -5,12 +5,25 @@ import matterAddImageEllipse from '@/helpers/matterAddImageEllipse';
 import Entity, { EntityConfigType } from '@/objects/entities/Entity';
 import keepUpright, { KeepUprightStratergies } from '@/helpers/keepUpright';
 import moveTowards from '@/helpers/moveTowards';
+import CollisionCategories from '@/enums/CollisionCategories';
+import useLocalStorage from '@/helpers/useLocalStorage';
 
 const KEY = 'ben3';
 
 const HEAD_SCALE_MIN = 0.1;
 const HEAD_SCALE_MAX = 0.5;
 
+// @ts-expect-error todo
+// Can't find correct type here? tried settting data to MatterJS.ICollisionPair but gameobject don't exsit on it.
+const onCollision = data => {
```

Comment:
Can't find correct type here? did try MatterJS.ICollisionPair but gameobject don't exist on it. I also tried 
` MatterJS.ICollisionPair & Entity`
` MatterJS.ICollisionPair & {gameobject: Entity}`

Reply:
it does work with:
```
type MyCollisionPair = MatterJS.ICollisionPair & {
  bodyB: { gameObject: Entity };
};
```
you were so close... :+1: 

then just need to update Entity.ts class var to public
```
public collisionCategory: CollisionCategories | undefined;
```

Reply:
Oooooh dam! I missed the BodyB param as well.... yeah that makes sense. Nice one 👍